src/components/AutoComplete/AutoCompleteV2.tsx

```diff
@@ -0,0 +1,276 @@
+import React, { memo, useState, ChangeEvent, KeyboardEvent, useEffect, useRef, useCallback } from 'react';
```

Comment:
I'd rename this to something more close to the job was that, like ... `AutoCompleteDropwDown` or something like that.

Reply:
sure thing!  I will do this in a follow-up PR ;-) 

but my idea here was to replace the current Autocomplete that we have..